### PR TITLE
Update according to clarification for rpc startservice

### PR DESCRIPTION
**Pull request by istoimenova**
Update is done according to clarification for StartService, servicetype = RPC

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/005_ATF_P_TC_default_app_first_StartService_RPC_encrypt_true.lua [hunk 1]
```diff
@@ -10,12 +10,11 @@
 -- (meaning: SDL will NOT respond to StartService_request from mobile app till PTU will be finished per comment)
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- Default app exists in LP, no certificate in module_config
 --
 -- 2. Performed steps
 -- 2.1. Register and activate application.
--- 2.2. Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- 2.2. Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/005_ATF_P_TC_default_app_first_StartService_RPC_encrypt_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 23

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/006_ATF_P_TC_comm_app_first_StartService_RPC_encrypt_true.lua [hunk 2]
```diff
@@ -10,12 +10,11 @@
 -- (meaning: SDL will NOT respond to StartService_request from mobile app till PTU will be finished per comment)
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- Communication app exists in LP, no certificate in module_config
 --
 -- 2. Performed steps
 -- 2.1. Register and activate application.
--- 2.2. Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- 2.2. Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/006_ATF_P_TC_comm_app_first_StartService_RPC_encrypt_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 23

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua [hunk 3]
```diff
@@ -10,12 +10,11 @@
 -- (meaning: SDL will NOT respond to StartService_request from mobile app till PTU will be finished per comment)
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- Media app exists in LP, no certificate in module_config
 --
 -- 2. Performed steps
 -- 2.1. Register and activate application.
--- 2.2. Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- 2.2. Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 23

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua [hunk 4]
```diff
@@ -11,11 +11,10 @@
 -- SDL must respond StartService (NACK) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to ON in .ini file
 -- Media app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 20

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/011_ATF_P_TC_comm_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua [hunk 5]
```diff
@@ -11,11 +11,10 @@
 -- SDL must respond StartService (NACK) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to ON in .ini file
 -- Communication app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/011_ATF_P_TC_comm_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 20

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/017_ATF_P_TC_media_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua [hunk 6]
```diff
@@ -11,18 +11,17 @@
 -- SDL must respond StartService (NACK) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to ON in .ini file
 -- Media app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/017_ATF_P_TC_media_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 20

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/018_ATF_P_TC_comm_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua [hunk 7]
```diff
@@ -11,18 +11,17 @@
 -- SDL must respond StartService (NACK) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to ON in .ini file
 -- Communication app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/018_ATF_P_TC_comm_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 20

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/019_ATF_P_TC_default_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua [hunk 8]
```diff
@@ -11,18 +11,17 @@
 -- SDL must respond StartService (NACK) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to ON in .ini file
 -- Default app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/019_ATF_P_TC_default_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 20

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/040_ATF_P_TC_default_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua [hunk 9]
```diff
@@ -11,11 +11,10 @@
 -- SDL must respond StartService (ACK, encrypted=false) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to OFF in .ini file
 -- Default app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/040_ATF_P_TC_default_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua]: @istoimenova, Please remove "SDL should not process request to HMI" statement in line 20

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua [hunk 10]
```diff
@@ -118,6 +115,7 @@ end
 commonFunctions:newTestCasesGroup("Postconditions")
 
 function Test.Postcondition_Restore_preloaded_pt()
+  commonPreconditions:RestoreFile("smartDeviceLink.ini")
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua]: @istoimenova, Please rename "Test.Postcondition_Restore_preloaded_pt" to "Test.Postcondition_Restore_Files"

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/005_ATF_P_TC_default_app_first_StartService_RPC_encrypt_true.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/005_ATF_P_TC_default_app_first_StartService_RPC_encrypt_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/006_ATF_P_TC_comm_app_first_StartService_RPC_encrypt_true.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/006_ATF_P_TC_comm_app_first_StartService_RPC_encrypt_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua (hunk 3 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua (hunk 10 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua (hunk 4 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/011_ATF_P_TC_comm_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua (hunk 5 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/011_ATF_P_TC_comm_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/017_ATF_P_TC_media_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua (hunk 6 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/017_ATF_P_TC_media_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/018_ATF_P_TC_comm_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua (hunk 7 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/018_ATF_P_TC_comm_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/019_ATF_P_TC_default_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua (hunk 8 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/019_ATF_P_TC_default_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/040_ATF_P_TC_default_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua (hunk 9 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/040_ATF_P_TC_default_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua]: @dboltovskyi, done.

**Pull request merged**